### DISCUSSION TITLE
Testing basic contract on truffle

### DISCUSSION THREAD

**Discussion by pbirbarah** [ConsenSys-archive/truffle#5178] [General Q&A]
this is my first contract and I am using truffle with ganache for testing and I am facing some issues:
![image](https://user-images.githubusercontent.com/74081842/172054759-86219ae6-04ce-49b0-b160-e4208342dae2.png)

The contract is supposed to just send payment back to the person who contacts it.
How do I do that? I have the contract deployed, created an instance const my_contract = await Test_Payment.deployed()

when I try my_contract.test I get an error that this is not a function I am confused how to proceed.

**Comment by kimanikelly** [comment]
Hi, @pbirbarah, do you have a public repository or full code snippet of Test_Payment.sol that can be referenced? I would like to assist you with the blocker you're facing, thank you.
